src/components/notification/Notification.js

```diff
@@ -8,6 +8,7 @@ import IconBell from '../../icons/general/IconBell';
 import IconClose from '../../icons/general/IconClose';
 import IconInfoThin from '../../icons/general/IconInfoThin';
 import IconSync from '../../icons/general/IconSync';
+import { KEYS } from '../../constants';
```

Comment:
If you dont mind, while you are at it, change #FFF to white imported from styles.js
line 123, 134